packages/types/package.json

```
@@ -31,7 +31,7 @@
     "bin/**/*.js"
   ],
   "dependencies": {
-    "@rdfjs/types": "*",
+    "@rdfjs/types": "^1.1.0",
```

```suggestion
    "@rdfjs/types": ">=1.1.0",
```
I'll just broaden the range a bit since this is a types package.